src/hardware/mod.rs

```diff
@@ -44,17 +43,22 @@ const RX_DESRING_CNT: usize = 4;
 pub type NetworkStack = smoltcp_nal::NetworkStack<
     'static,
     hal::ethernet::EthernetDMA<'static, TX_DESRING_CNT, RX_DESRING_CNT>,
-    system_timer::SystemTimer,
+    SystemTimer,
 >;
 
 pub type NetworkManager = smoltcp_nal::shared::NetworkManager<
     'static,
     hal::ethernet::EthernetDMA<'static, TX_DESRING_CNT, RX_DESRING_CNT>,
-    system_timer::SystemTimer,
+    SystemTimer,
 >;
 
 pub type EthernetPhy = hal::ethernet::phy::LAN8742A<hal::ethernet::EthernetMAC>;
 
+/// System timer (RTIC Monotonic) tick frequency
+pub const HZ: u32 = 1_000;
+pub type Systick = systick_monotonic::Systick<HZ>;
+pub type SystemTimer = mono_clock::MonoClock<u32, HZ>;
```

Comment:
```suggestion
pub const MONOTONIC_FREQUENCY: u32 = 1_000;
pub type Systick = systick_monotonic::Systick<MONOTONIC_FREQUENCY>;
pub type SystemTimer = mono_clock::MonoClock<u32, MONOTONIC_FREQUENCY>;
```

I'm not a fan of just `HZ` due to its vagueness